src/condor_utils/submit_utils.cpp

```diff
@@ -6345,6 +6346,7 @@ int SubmitHash::process_container_input_files(StringList & input_files, long lon
 		StringList roots(sharedfs.ptr(), ",");
 		for (const char * base = roots.first(); base != NULL; base = roots.next()) {
 			if (starts_with(container_image.ptr(), base)) {
+				job->Assign(ATTR_CONTAINER_IMAGE_SOURCE, "local");
```

Comment:
if it's on a shared file system, is it really local?

Reply:
Yes, from the purpose of the AP.